src/components/formreporttype.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
+import * as Yup from "yup";
+import ErrorMsg from "./errormsg";
+
+const getSubCategories = (categories, categoryId) => {
+  var cats = categories.find(category => category.id === categoryId);
+  return cats ? cats.types : [];
+};
+
+const ServiceRequestForm = props => {
+  const [categories] = useState(Categories);
+  const [subCategories, setSubCategories] = useState([]);
+
+  const handleServiceRequestChange = changeEvent => {
+    const { value } = changeEvent.currentTarget;
+    const subCategories = getSubCategories(categories, parseInt(value));
+    setSubCategories(subCategories);
+  };
+
+  return (
+    <React.Fragment>
+      <div className="container Container-bg">
```

Comment:
Can we create a compoent called something like FormContainer which looks something like

```js
const FormContainer = props => <div className="container Container-bg"><h3>{props.title}</h3>{props.children}</div>"
```

Then this compoent should look like

```js
return (
  <FormContainer title="How Can We Help?">
    <Formik .... /> 
  </FormContainer>
);

Reply:
FormContainer created